packages/components/addon/components/hds/pagination-bar/pagination.hbs

```diff
@@ -3,60 +3,74 @@
   aria-label="Pagination navigation"
   ...attributes
 >
-  <a class="hds-pagination__page-link" href="/page-?" aria-label="Previous page">
+  <Hds::Interactive
```

Comment:
Probably I would abstract this straight away in its own component, so I can:
- re-use it/share it between `prev` and `next`
- test it in isolation (the API, the UI, the interaction)
- add it in the showcase, with its different variants and states

Reply:
Refactors are still a WIP but I separated out the buttons now and pushed that up. I will work on updating the showcase, etc. later on.